README.md

```diff
@@ -495,6 +495,30 @@ Example using cURL:
 curl -X DELETE localhost:5000/services/my-service
 ```
 
+#### DELETE /scopes?env=alpha&dry_run
+
+Delete scope entries using regex to match on keys that you wish to delete. A request body with the 'regexString' field is required. 'regexString' must be a non-empty string. Using the 'dry_run' query parameter will return the entries that will be deleted; not affecting the actual import map. Don't forget to escape backslash's in your regexString if they are to literally be apart of the regex... This is mandatory for JSON parsers.
```

Comment:
Why is the regexString being sent in the request body rather than encoded in the url path? I first expected to see `DELETE /scopes/:urlEncodedScope` as the API rather than a request body. I've never used request bodies with the HTTP DELETE method before, and some sources (including [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Methods/DELETE)) suggest that DELETE requests shouldn't have a request body.